tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "target": "ES2021",
     "module": "ES2020",
     "moduleResolution": "node",
+    "skipLibCheck": true,
```

Comment:
How is this related to the change?

Reply:
Without it there's an type check error:

```
[test:ember] node_modules/.pnpm/@types+eslint@7.29.0/node_modules/@types/eslint/helpers.d.ts:1:6 - error TS2300: Duplicate identifier 'Prepend'.
[test:ember]
[test:ember] 1 type Prepend<Tuple extends any[], Addend> = ((_: Addend, ..._1: Tuple) => any) extends (..._: infer Result) => any
[test:ember]        ~~~~~~~
[test:ember]
[test:ember]   node_modules/.pnpm/@types+eslint@8.37.0/node_modules/@types/eslint/helpers.d.ts:1:6
[test:ember]     1 type Prepend<Tuple extends any[], Addend> = ((_: Addend, ..._1: Tuple) => any) extends (..._: infer Result) => any
[test:ember]            ~~~~~~~
[test:ember]     'Prepend' was also declared here.
[test:ember]
[test:ember] node_modules/.pnpm/@types+eslint@8.37.0/node_modules/@types/eslint/helpers.d.ts:1:6 - error TS2300: Duplicate identifier 'Prepend'.
[test:ember]
[test:ember] 1 type Prepend<Tuple extends any[], Addend> = ((_: Addend, ..._1: Tuple) => any) extends (..._: infer Result) => any
[test:ember]        ~~~~~~~
[test:ember]
[test:ember]   node_modules/.pnpm/@types+eslint@7.29.0/node_modules/@types/eslint/helpers.d.ts:1:6
[test:ember]     1 type Prepend<Tuple extends any[], Addend> = ((_: Addend, ..._1: Tuple) => any) extends (..._: infer Result) => any
[test:ember]            ~~~~~~~
[test:ember]     'Prepend' was also declared here.
```

But maybe it can be fixed differently

Reply:
Can we try adding 

```js
{
  "exclude": [
    "node_modules"
  ]
}
```

Reply:
No, exclude doesn't resolve these errors. Maybe we can add a resolution for `@types/eslint` version, WDYT?

Reply:
When do you see the type check error?

Reply:
Let's keep it for now until we have a solution for this.  Keeping an eye in here https://github.com/typed-ember/ember-cli-typescript/issues/1502